ext/intl/grapheme/grapheme_string.c

```diff
@@ -1053,8 +1053,8 @@ PHP_FUNCTION(grapheme_levenshtein)
 	}
 
 	zend_long *p1, *p2, *tmp;
-	p1 = safe_emalloc(strlen_2 + 1, sizeof(zend_long), 0);
-	p2 = safe_emalloc(strlen_2 + 1, sizeof(zend_long), 0);
+	p1 = safe_emalloc((size_t) strlen_2 + 1, sizeof(zend_long), 0);
```

Comment:
Why not to declare `strlen_1` of `size_t` type directly? Can this function work 4GB+ length string?

Reply:
Because if you declare it as unsigned, the error check won't work